packages/suite/src/components/suite/DeviceDisplay.tsx

```diff
@@ -0,0 +1,230 @@
+    gap: ${props => (props.isPixelType ? '6' : '5')}px;
+`;
+
+const Wrapper = styled.div`
+    position: relative;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+`;
+
+const Divider = styled.div<{ isPixelType: boolean }>`
+    width: 100%;
+    height: 1px;
+    background: #2b2b2b;
+    margin: ${props => (props.isPixelType ? '10px' : '0')} 0;
+`;
+
+const AddressLabel = styled.span<{ isPixelType: boolean }>`
+    font-weight: 600;
+    color: #868686;
+    font-size: ${variables.FONT_SIZE.TINY};
+    text-transform: uppercase;
+    position: absolute;
+    background: #000000;
+    padding: 0 10px;
+    text-align: center;
+    left: 50%;
+    bottom: ${props => (props.isPixelType ? '2' : '-7')}px;
+    transform: translate(-50%, 0);
+`;
+
+const StyledNextIcon = styled(Icon)<{ isPixelType: boolean }>`
+    position: relative;
+    bottom: ${props => (props.isPixelType ? '13' : '20')}px;
+    right: ${props => (props.isPixelType ? '-95' : '35')}px;
+`;
+
+const StyledContinuesIcon = styled(Icon)<{ isPixelType: boolean }>`
+    position: relative;
+    top: ${props => (props.isPixelType ? '10' : '25')}px;
+    right: ${props => (props.isPixelType ? '82' : '97')}px;
+`;
+
+const StyledChunkedNextIcon = styled(Icon)<{ isPixelType: boolean }>`
+    position: relative;
+    bottom: ${props => (props.isPixelType ? '13' : '20')}px;
+    right: -67px;
+`;
+
+const StyledChunkedContinuesIcon = styled(Icon)<{ isPixelType: boolean }>`
+    position: relative;
+    top: ${props => (props.isPixelType ? '10' : '25')}px;
+    right: 68px;
+`;
+
+export interface DeviceDisplayProps {
+    address: string;
+    network?: string;
+}
+
+export const DeviceDisplay = ({ address, network }: DeviceDisplayProps) => {
+    const selectedDeviceInternalModel = useSelector(selectDeviceInternalModel);
+    const unavailableCapabilities = useSelector(selectDeviceUnavailableCapabilities);
+    const addressDisplay = useSelector(selectAddressDisplay);
+
+    const areChunksUsed =
+        addressDisplay === AddressDisplayOptions.CHUNKED && !unavailableCapabilities?.chunkify;
+    const isPixelType = selectedDeviceInternalModel !== DeviceModelInternal.T2T1;
+    const isPaginated = network === 'cardano';
+
+    const showChunksInRows = (chunks: string[][] | undefined, isNextAddress?: boolean) =>
+        chunks?.map((row, rowIndex) => (
+            <Row
+                // eslint-disable-next-line react/no-array-index-key
+                key={rowIndex}
+                isPixelType={isPixelType}
+                isAlignedRight={rowIndex === 0 && isNextAddress}
+            >
+                {row.map((chunk, chunkIndex) => (
+                    // eslint-disable-next-line react/no-array-index-key
+                    <Text isPixelType={isPixelType} key={chunkIndex}>
+                        {chunk}
+                    </Text>
+                ))}
+            </Row>
+        ));
+
+    const showChunks = (address: string) => {
```

Comment:
nit: `getChunks`

Reply:
I think `renderChunks` that you suggested later is better